Base/Python/sitkUtils.py

```diff
@@ -146,7 +150,7 @@ def sitk2vtk(img):
     vtk_image.SetOrigin(0.0, 0.0, 0.0)
 
     # Set pixel data
-    depth_array = vtknp.numpy_to_vtk(npdata.ravel())
+    depth_array = vtknp.numpy_to_vtk(npdata.ravel(), deep=deep)
```

Comment:
Are there only two possible states - deep or shallow? If yes, then we could change this to something more explicit, such as 1 if deepCopy else 0